site/content/docs/tutorials/concierge-and-supervisor-demo.md

```diff
@@ -1,239 +1,654 @@
+Save the admin kubeconfig files somewhere private and secure for your own future use.
+
+See the [full documentation for the `pinniped get kubeconfig` command]({{< ref "../reference/cli" >}})
+for other available optional parameters.
+
+### As a developer or devops user, access the workload clusters by using regular kubectl commands
+
+A developer or devops user who would like to use the workload clusters may do so using kubectl with
+the kubeconfig files provided to them by the cluster admin in the previous step.
+
+The kubeconfig files tell kubectl how to invoke the Pinniped CLI as a plugin to aid in authentication.
+First, the user will need to install the Pinniped CLI at the same full path where it is referenced
+inside the kubeconfig file. Or, they can adjust the full path to the Pinniped CLI inside
+their own copy of the kubeconfig file, to make it match where they have locally installed the Pinniped CLI.
+
+Then the developer can run any kubectl command using a kubeconfig file
+that was provided to them by the cluster admin. For example, let's run a command against the first workload cluster.
+
+```sh
+kubectl get namespaces --kubeconfig workload1-developer.yaml
+```
+
+The first time this command is run, it will open their default web browser and redirect them to Okta for login.
+After successfully logging in to Okta, for example as the user `walrus@example.com`, the kubectl command will
+continue and will list the namespaces.
+The user's identity in Kubernetes (username and group memberships) came from Okta, through Pinniped.
+
+That same developer user can access all other workload clusters in a similar fashion. For example,
+let's run a command against the second workload cluster.
+
+```sh
+kubectl get namespaces --kubeconfig workload2-developer.yaml
+```
+
+This time, the command will list namespaces immediately.
+Even though you are accessing a different cluster, the web browser will not open again.
+You do not need to interactively sign in again for the rest of the day to access
+any workload cluster within the same FederationDomain.
+Behind the scenes, Pinniped is performing token refreshes and token exchanges
+on behalf of the user to create a short-lived, cluster-scoped token to access
+this new workload cluster using the same identity from Okta.
+
+If the user did not have RBAC permissions to perform the requested action, then they would see an error
+from kubectl similar to
+`Error from server (Forbidden): namespaces is forbidden: User "walrus@example.com" cannot list resource "namespaces" in API group "" `.
+
+Note that users can use any of kubectl's supported means of providing kubeconfig information to kubectl.
+They are not limited to only using the `--kubeconfig` flag. For example, they could set the `KUBECONFIG`
+environment variable instead.
+
+For more information about logging in to workload clusters, see the [howto doc about login]({{< ref "../howto/login" >}}).
+
```

Comment:
Maybe add an example of how to merge these kubeconfig files into a single kubeconfig

Reply:
Will do.